src/rapidfuzz/process.py

```diff
@@ -10,9 +10,9 @@ def iterExtract(query: str, choices: Iterable, scorer: Callable = fuzz.WRatio, p
             score_cutoff: float = 0) -> Generator[Tuple[str, float], None, None]:
     a = processor(query) if processor else query
 
-    if isinstance(choices, list):
-        for choice in choices:
-            b = processor(choice) if processor else choice
+    try:
+        for choice, match_choice in choices.items():
+            b = processor(match_choice) if processor else match_choice
```

Comment:
I think it might be better to check `if hasattr(choices, "items")` simply because I think the default case is having a iterable and not a dict -> the exception is raised for most people

Reply:
I did a simple benchmark, and it appears that your solution is twice as fast (although we're speaking in milliseconds of difference). So I'll change the implementation.

Reply:
Yes this is definetly not a big thing since more than 99% of the runtime will be inside the inner for loop anyways :)